substrate-node/pallets/pallet-tft-bridge/src/types.rs

```diff
@@ -2,6 +2,19 @@ use parity_scale_codec::{Decode, Encode};
 use scale_info::TypeInfo;
 use sp_std::vec::Vec;
 
+/// Utility type for managing upgrades/migrations.
+#[derive(Encode, Decode, Clone, Debug, PartialEq, PartialOrd, TypeInfo)]
+pub enum StorageVersion {
+    V1,
+    V2, // add source to both BurnTransaction type and ExpiredBurnTransaction event
+}
+
+impl Default for StorageVersion {
+    fn default() -> StorageVersion {
+        StorageVersion::V1
```

Comment:
Since the pallet has no version stored, then it will default to v2. This means that the pallet version will return v2 when checked, and the migration will not run.